### ISSUE TITLE
custom control over notification/bluetooth control

### ISSUE THREAD

**Issue by laurnt** [florent37/Flutter-AssetsAudioPlayer#120]
Hi, not sure if this is already possible but for my application I use long audio tracks and simulate radio by setting seek based on the time of day. This works fine with my own audio controls but not with the notification controls. Is there any way to assign functions to the notification buttons?

**Comment by florent37** [comment]
I'm currently working on it :)

**Comment by florent37** [comment]
I linked the pull request
but I don't wanna make the buttons icons mutable because of ios, on ios the media center only have prev/play-pause/next, only on android the icon are mutable

**Comment by florent37** [comment]
but it will invoke a custom lambda on click with this pull request

**Comment by florent37** [comment]
published in `1.6.2+1`


```dart
open(AUDIO,
   showNotification: true,
   notificationSettings: NotificationSettings(
       prevEnabled: false, //disable the previous button
  
       //and have a custom next action (will disable the default action)
       customNextAction: (player) {
         print("next");
       }
   )

)
```
